DataCenterBridging.psm1

```diff
@@ -638,6 +638,70 @@ function Get-LldpPfc
 }
 
 
+function Test-LbfoEnabled

```

Comment:
Would it be better to use the existing logic flow where we detect whether teamed or not, then add new logic to look up the bindings with Get-NetAdapterBinding? This would tell us in significantly less complexity (i think) if we have an LBFO team.

Reply:
I'll look into that for the next revision. Didn't think of looking at the bindings to detect LBFO.